block-commits/action.yml

```diff
@@ -0,0 +1,19 @@
+name: "Block Merge and Autosquash Commits"
+description: "Prevent merging pull requests that contain merge commits and commits with messages prefixed with fixup! or squash!"
```

Comment:
https://github.com/trinodb/trino/pull/13494#issuecomment-1206877671

drop "and Autosquash" from here

Reply:
oh, is it optional?

Reply:
Yes, it's configurable.